Commencement.Mvc/Controllers/TicketController.cs

```diff
@@ -21,6 +21,7 @@ public class TicketController : ApplicationController
         public ActionResult Index()
         {
             var term = TermService.GetCurrent();
+            //var xxx = term.Ceremonies.ToList(); //Scott, this is generating a Lazy Load error

```

Comment:
@srkirkland When you get back, lets go over this. Not sure why it generates an error. I fixed it by getting the ceremonies directly.